conf/policy.json

```diff
@@ -51,5 +51,15 @@
   "metrics:collect": "rule:admin_or_owner",
   "metrics:uploadconf": "rule:admin_api",
   "metrics:downloadconf": "rule:admin_api",
-  "metrics:urls": "rule:admin_or_owner"
+  "metrics:urls": "rule:admin_or_owner",
+  "fileshare:create": "rule:admin_or_owner",
+  "fileshare:delete": "rule:admin_or_owner",
+  "fileshare:list": "rule:admin_or_owner",
+  "fileshare:get": "rule:admin_or_owner",
+  "fileshare:update": "rule:admin_or_owner",
+  "fileshareacl:create": "rule:admin_or_owner",
+  "fileshareacl:delete": "rule:admin_or_owner",
+  "fileshareacl:list": "rule:admin_or_owner",
+  "fileshareacl:get": "rule:admin_or_owner",
+  "fileshareacl:update": "rule:admin_or_owner"

```

Comment:
Please add file share snapshot operations.

Reply:
in policy.json, File and volume use snapshot permissions together, do we don't have to add file snapshot here.